scripts/swift_lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+export PATH="$PATH:/opt/homebrew/bin"
+if which swiftlint > /dev/null; then
+  swiftlint
```

Comment:
Should this be swiftformat?

Reply:
This was because i merged the swift-lint PR into this after I addressed your comment about relocating swiftlint into a designated bash script